src/code/z_actor.c

```diff
@@ -1768,7 +1768,7 @@ u32 Actor_TextboxIsClosing(Actor* actor, PlayState* play) {
     }
 }
 
-s8 func_8002F368(PlayState* play) {
+s8 Actor_GetPlayerExchangeItemId(PlayState* play) {
     Player* player = GET_PLAYER(play);
 
     return player->exchangeItemId;
```

Comment:
I was thinking maybe `Actor_GetOfferedExchangeItem` or something. As in, get the exchange item that player is offering to the actor.

Dont have to do with this one right away, am curious on other opinions.

Reply:
I slightly prefer "Player" to "Offered", I feel it makes me think less. Dunno about dropping `Id`, it seems inconsistent rn (e.g. `EXCH_ITEM_ODD_MUSHROOM` and not `EXCH_ITEM_ID_ODD_MUSHROOM`) so idk

Reply:
so, the name as it is now is what you prefer?

Reply:
Either `Actor_GetPlayerExchangeItemId` or `Actor_GetPlayerExchangeItem` I guess, but I don't really care